blueprints.yaml

```diff
@@ -23,66 +23,81 @@ form:
         0: Disabled
       validate:
         type: bool
```

Comment:
most of the users don't need translations and would like to customize the cookie consent texts over the admin interface. Also it would breaks compatibility to older versions. Therefore I would like to keep `content_message`, `content_dismiss`, `content_link` and `content_href`.

Reply:
I remove them because it can be done with translation files. But I understand your point. Of course you can put them back